_posts/2018-12-11-Q-learning.markdown

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title:  "Introduction to Q-learning!"
+date:   2018-12-11 11:42:04 +0530
+categories: jekyll update
+author: Madhuparna Bhowmik
+---
+
+In this article, we will learn about Q-learning and implement the algorithm from scratch. We will also make our own small board game to test the algorithm.
+### **What is Q-learning**
```

Comment:
'**' is not required since you already are making it a subheading with '###'. Also, change it to '##'.